server/log_rpc_server_test.go

```diff
@@ -1278,20 +1285,21 @@ func TestGetConsistencyProof(t *testing.T) {
 		response, err := server.GetConsistencyProof(context.Background(), &test.req)
 
 		if test.wantErr {
-			if err == nil {
-				t.Errorf("GetConsistencyProof(%+v)=nil, want: err", test.req)
+			if err == nil || !strings.Contains(err.Error(), test.errStr) {
+				t.Errorf("GetConsistencyProof(%+v)=_,nil, want: _,err contains: %s but got: %v", test.req, test.errStr, err)
```

Comment:
the error message is now inaccurate -- something like:
```
t.Errorf("GetConsistencyProof(%+v)=_, %v; want _, err containing %q", test.req, err, test.errStr)
```
would cover both cases

Reply:
Done.